3.-Web-api-call-Microsoft-graph-for-personal-accounts/README.md

```diff
@@ -172,64 +165,28 @@ If you want to register your apps manually, as a first step you'll need to:
      - in **User consent description** type `Accesses the TodoListService Web API as a user`
      - Keep **State** as **Enabled**
      - Select **Add scope**
-1. [Optional] Select the **Manifest** section and:
-   - in the manifest, search for **"accessTokenAcceptedVersion"**, and see that its value is **2**. This property lets Azure AD know that the Web API accepts v2.0 tokens
-   - Select **Save**
 
-> Important: it's up to the Web API to decide which version of token (v1.0 or v2.0) it accepts. Then when clients request a token for your Web API using the v2.0 endpoint, they'll get a token which version is accepted by the Web API. The code validating the tokens in this sample was written to accept both versions.
+#### Register the client part (in the same app)
```

Comment:
>part [](start = 25, length = 4)

nit: not a fan of "part"...i would just remove it everywhere and use client and service, or maybe use "component" instead? idk.

Reply:
I used aspect. Please tell me if this is still wrong.

Reply:
looks good.

---
In reply to: [287955385](https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/pull/53#discussion_r287955385) [](ancestors = 287955385)